### DISCUSSION TITLE
Cant figure out what is the issues! ESPpanel has no connection.

### DISCUSSION THREAD

**Discussion by RealErnie** [Blackymas/NSPanel_HA_Blueprint#2420] [General]
Hi,
i searched over the web and cant figure out what the problem is about!

guess no connection, but i have no idea to fix.

regards

![IMG_20241201_112709](https://github.com/user-attachments/assets/d4fa65ec-7d6f-4034-a766-712650e3e70b)

**Comment by edwardtfn** [comment]
Some logs would be useful, but looks like your panel isn't communicating with the Blueprint.
Could you please open the automation (which uses the Blueprint) and make sure the panel is selected?
